include/gatekeeper_ggu.h

```diff
@@ -96,36 +167,12 @@ struct ggu_policy {
 		 * struct flow_entry at the corresponding state.
 		 */
 		union {
-			struct {
-				/* Rate limit: kilobyte/second. */
-				uint32_t tx_rate_kb_sec;
-				/*
-				 * How much time (unit: second) a GK block waits
-				 * before it expires the capability.
-				 */
-				uint32_t cap_expire_sec;
-				/*
-				 * The first value of send_next_renewal_at at
-				 * flow entry comes from next_renewal_ms.
-				 */
-				uint32_t next_renewal_ms;
-				/*
-				 * How many milliseconds (unit) GK must wait
-				 * before sending the next capability renewal
-				 * request.
-				 */
-				uint32_t renewal_step_ms;
-			} granted;
-
-			struct {
-				/*
-				 * How much time (unit: second) a GK block waits
-				 * before it expires the declined capability.
-				 */
-				uint32_t expire_sec;
-			} declined;
+			/* Decision is to grant the flow. */
+			struct ggu_granted granted;
+			/* Decision is to decline the flow. */
+			struct ggu_declined declined;
 		} u;
-	}__attribute__((packed)) params;
+	} __attribute__((packed)) params;
```

Comment:
The new `struct ggu_granted` and `struct ggu_declined` are already packed, so we could drop the whole `struct` around the `union` here to simplify the code.